apps/docs/pages/guides/database/large-datasets.mdx

```diff
@@ -0,0 +1,121 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'database',
+  title: 'Importing into Supabase',
+  description: 'Importing Large Datasets into Supabase.',
+  subtitle: 'Importing Large Datasets into Supabase.',
+  sidebar_label: 'FUNDAMENTALS',
+}
+
+Importing large datasets requires careful planning and execution to ensure the process is efficient, reliable, and doesn't disrupt your application. In this guide, we cover various best practices and techniques for importing datasets, while also highlighting common pitfalls to avoid.
+
+## **You have multiple options for importing your data into Supabase**
+
+### Option 1: CSV Import via Supabase Dashboard
+
+    Supabase dashboard provides a user-friendly way to import data. However, for very large datasets, this method may not be the most efficient choice, given the size limit is 100MB. It's generally better suited for smaller datasets and quick data imports. Consider using alternative methods like pgloader for large-scale data imports.
+
+    - Navigate to the relevant table in the [Table Editor.](https://supabase.com/dashboard/project/_/editor)
+    - Click on “Insert” then choose "Import Data from CSV" and follow the on-screen instructions to upload your CSV file.
+
+### Option 2: Bulk Import using pgloader
+
+    [pgloader](https://pgloader.io/) is a powerful tool for efficiently importing data into a PostgreSQL database that supports a wide range of source database engines, including MySQL and MS SQL.
+
+    You can use it in conjunction with Supabase by following these steps:
```

Comment:
There's a bunch of empty spaces at the beginning of these sentences it seems, could you remove them please?